manager/src/main/java/com/redhat/developer/manager/models/ConnectorStatus.java

```diff
@@ -0,0 +1,41 @@
+package com.redhat.developer.manager.models;
+
+import com.redhat.developer.infra.dto.ConnectorStatusDTO;
+
+public enum ConnectorStatus {
+    REQUESTED("REQUESTED"),
+    PROVISIONING("PROVISIONING"),
+    AVAILABLE("AVAILABLE");
+
+    private String status;
+
+    ConnectorStatus(String status) {
+        this.status = status;
+    }
+
+    public ConnectorStatusDTO toDTO() {
```

Comment:
This feels like a bit of a code smell...is there a reason we can't use a single enum?